cmd/quadlet/main.go

```diff
@@ -210,6 +211,66 @@ func enableServiceFile(outputPath string, service *parser.UnitFile) {
 	}
 }
 
+func isSha256Digest(imageName string) bool {
+	if strings.HasPrefix(imageName, "sha256:") {
+		return true
+	}
+
+	// only fully specified digests are matched
+	if len(imageName) != 64 {
+		return false
+	}
+
+	for _, c := range imageName {
+		if !unicode.Is(unicode.Hex_Digit, c) {
```

Comment:
I’d also expect this to fail on the `sha256:` prefix.

Reply:
Again, the goal isn't to validate that this is a valid reference, podman run already does that. the goal is just to avoid triggering a shortname lookup. So, I'm not sure its super useful to add more validation into this warning.

Reply:
I was mistaken about what this function intends to do, see elsewhere.